doc/index.rst

```diff
@@ -254,6 +258,35 @@ The jobs that run Prerelease Test may usually take longer than the tests defined
 
 Then open a pull request using this branch against the branch that the change is subject to be merged. You do not want to actually merge this branch no matter what the Travis result is. This branch is solely for Prerelease Test purpose.
 
+ABI checks
+----------
+
+Generally speaking, the `ABI <https://en.wikipedia.org/wiki/Application_binary_interface>`_ of a library might break for various reasons. A detailed explanation and a list of DOs and DON'Ts can be found in the `KDE Community Wiki <https://community.kde.org/Policies/Binary_Compatibility_Issues_With_C%2B%2B>`_.
+
+The ABI checks with `industrial_ci` can be enabled by setting 'ABICHECK_URL' to a code archive with your stable version (e.g. tagged version or older tagged versions of your package)
+The following is a few examples of those URL:
+
+  - https://github.com/ros-planning/moveit/archive/0.9.9.tar.gz
+  - https://gitlab.com/YOUR_ORGANIZATION/YOUR_REPO/repository/ver_0.0.8/archive.zip
+  - https://github.com/ros-industrial-release/ros_canopen-release/archive/upstream.zip
+  - github:ros-planning/moveit#0.9.9
+
+The last example shows the shortcut from `provider:organization/repository#version`, which is supported for bitbucket, github and gitlab.
+
+ABI check examples:
```

Comment:
Below there are only `ssh`-based URL examples available, while in the URL example above there are more `http` URLs. This could give an impression that http is more common IMO, and we should have usecase example that uses http in addition.

(Wish I could open another PR for a small fix like this but today my wifi is very bad.)

Reply:
Where do we have "ssh-based URL examples"? Only curl-compatible protocols (http(s), ftp?) can be used.
The shortcut syntax is based on `npn install` URLs.

Reply:
I thought the one that starts from `git`, i.e. `github:ros-planning/moveit#0.9.9`, was an ssh format, but it isn't. 

What is `npn install`?

Reply:
I'm not familiar with this format `github:ros-planning/moveit#0.9.9` at all. Don't know where to get that for packages I'm willing to run abi check for.
Could you also document about this format?

Reply:
> What is npn install?

Sry, I misspelled it. `npm` is the packages manager for nodejs.

> I'm not familiar with this format github:ros-planning/moveit#0.9.9 at all. Don't know where to get that for packages I'm willing to run abi check for.
Could you also document about this format?

It is documented in line 274, you just have to fill in the parts.
`github:ros-planning/moveit#0.9.9` will be resolved to `https://github.com/ros-planning/moveit/archive/0.9.9.tar.gz`, it is just shorter.

(Note to self: fix typo in line 274 from -> form)